### ISSUE TITLE
Push to github doesn't work

### ISSUE THREAD

**Issue by boka18** [penxio/penx#26]
It either doesn't work, or I don't know how to use it. Here is what I did:
1. Created new repo 
2. Connected my github acc with `penx`
3. click on push `push to github`
4. got `sync failed` with this errs
```
sse error=============: TypeError: NetworkError when attempting to fetch resource. [runSSE.ts:56:14](webpack://_N_E/packages/app/src/common/runSSE.ts?4843)
==========github push error: HttpError: Git Repository is empty.
```

Is there something I'm missing here?

**Comment by steveschult** [comment]
Have you created the repository and initial committed it? The repository must have an initial commit

**Comment by boka18** [comment]
creates a repo, but it's empty. what should I commit to it or it doesn't matter as long as it has initial commit?

**Comment by steveschult** [comment]
Yes,It is convenient to Initialize this repository with: Add a README file. 
<img width="763" alt="37" src="https://github.com/penxio/penx/assets/146011744/6394f841-f3e0-433c-8c48-d12886a19d60">

**Comment by steveschult** [comment]
I just tried it. There is still a bug in github synchronization. It will be fixed later.
